src/components/Explore/NowPlaying.tsx

```diff
@@ -0,0 +1,50 @@
+import { css } from "@emotion/css";
+import React from "react";
+import { fetchLatestTracks } from "../../services/Api";
+import usePagination from "../../utils/usePagination";
+import { CenteredSpinner } from "../common/Spinner";
+import TrackList from "../common/TrackList";
+
+export const NowPlaying: React.FC = () => {
+  const [tracks, setTracks] = React.useState<Track[]>([]);
+
+  const { results, isLoading } = usePagination<Track>({
+    apiCall: React.useCallback(fetchLatestTracks, []),
+    options: { limit: 2, order: "plays" },
+  });
+
+  React.useEffect(() => {
+    setTracks(
+      results.map((r) => ({
+        ...r,
+        images: {
+          ...r.images,
+          small: {
+            // FIXME: There's a bug with the API's `images` being returned
+            // from fetchTracks. The URLs seem to not point to any existing files.
+            url: r.cover,
+            width: 120,
+            height: 120,
+          },
+        },
+      }))
+    );
+  }, [results]);
+
+  return (
+    <>
+      <h3>Now Playing</h3>
+      <p
+        className={css`
+          margin-bottom: 1rem;
+        `}
+      >
+        What's currently playing in the Resonate ecosystem?
+      </p>
+      {isLoading && <CenteredSpinner />}
+      <TrackList tracks={tracks} draggable />
```

Comment:
I'm not sure `draggable` makes sense here because the library sidebar isn't available right?